pkg/storage/image.go

```diff
@@ -39,19 +39,30 @@ type ImageResult struct {
 	Size         *uint64
 	Digest       digest.Digest
 	ConfigDigest digest.Digest
+	User         string
 }
 
 type indexInfo struct {
 	name   string
 	secure bool
 }
 
+// A set of information that we prefer to cache about images, so that we can
+// avoid having to reread them every time we need to return information about
+// images.
+type imageCacheItem struct {
+	user         string
+	size         *uint64
+	configDigest digest.Digest
+}
```

Comment:
I don't think they will change, but want to ask.  Is it possible for these values to change for an image once they're initially set?  Otherwise just wondering if an image could remain in the cache and one or more of these values change outside of the cache.  I'm 99% sure it won't be an issue, but that 1% has a tendency to bite me if I don't ask.

Reply:
The storage library doesn't provide an API for changing the IDs of things, so they should be fixed from when we first create the image record to when it's deleted.